src/Network/Network/NetworkManager/NetworkGroup/NewAzNetworkGroupCommand.cs

```diff
@@ -101,11 +111,16 @@ private PSNetworkManagerGroup CreateNetworkGroup()
                 psNetworkGroup.Description = this.Description;
             }
 
+            if (!string.IsNullOrEmpty(MemberType))
+            {
+                psNetworkGroup.MemberType = this.MemberType;
```

Comment:
we defult to vnet on backend, correct? if so, describe that in the md docs

Reply:
Updated.